cognitive/src/test/scala/com/microsoft/azure/synapse/ml/cognitive/anomaly/MultivariateAnamolyDetectionSuite.scala

```diff
@@ -264,3 +270,67 @@ class SimpleFitMultivariateAnomalySuite extends EstimatorFuzzing[SimpleFitMultiv
 
   override def modelReader: MLReadable[_] = SimpleDetectMultivariateAnomaly
 }
+
+class DetectLastMultivariateAnomalySuite extends TransformerFuzzing[DetectLastMultivariateAnomaly]
+  with MADTestUtils {
+
+  val hc: Configuration = spark.sparkContext.hadoopConfiguration
+  hc.set("fs.azure", "org.apache.hadoop.fs.azure.NativeAzureFileSystem")
+  hc.set(s"fs.azure.account.keyprovider.$storageAccount.blob.core.windows.net",
+    "org.apache.hadoop.fs.azure.SimpleKeyProvider")
+  hc.set(s"fs.azure.account.key.$storageAccount.blob.core.windows.net", storageKey)
+
+  val sfma: SimpleFitMultivariateAnomaly = new SimpleFitMultivariateAnomaly()
+    .setSubscriptionKey(anomalyKey)
+    .setLocation(anomalyLocation)
+    .setOutputCol("result")
+    .setStartTime(startTime)
+    .setEndTime(endTime)
+    .setIntermediateSaveDir(intermediateSaveDir)
+    .setTimestampCol(timestampColumn)
+    .setInputCols(inputColumns)
+    .setSlidingWindow(50)
+
+  val model: SimpleDetectMultivariateAnomaly = sfma.fit(df)
+  val modelId: String = model.getModelId
+
+  MADUtils.CreatedModels += modelId
```

Comment:
make this stuff lazy, add the   

lazy val modelId = {
  val model = sfma.fit(df)
  MADUtils.CreatedModels += model.getModelId
  model.getModelId
}